frontend/apps/reader/modules/readerbookmark.lua

```diff
@@ -448,11 +448,11 @@ function ReaderBookmark:onShowBookmark(open_navigator)
             page = bookmark.ui.document:getPageFromXPointer(ipage)
         end
         if page ~= "" then
-            title = title .. "  -  " .. _("page") .. " " .. page
+            title = T("%1  -  %2 %3", title, _("page"), page)
```

Comment:
Probably not really good, for languages that would display it like "4'paged". Best to include the whole %1 %2 %3 in a _ template, and have T(_(template), value1, value2...).
with a comment above like:
`-- @translators %1 is the bookmark menu title with index. %2 is the page number`

Or `title .. " - " .. T(_("page %1"), page)` if you think like I do that no translator would want to re-order title and page.

Reply:
@poire-z In America they say today's September 15, 2020.